PowerEditor/src/ScintillaComponent/Buffer.cpp

```diff
@@ -1483,21 +1483,23 @@ bool FileManager::loadFileData(Document doc, int64_t fileSize, const TCHAR * fil
 		}
 		else // x64
 		{
+			NppGUI& nppGui = NppParameters::getInstance().getNppGUI();
+			if (!nppGui._suppress2GBWarning) {

```

Comment:
Please respect coding convention in Notepad++ :
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#general